bitcoin/src/sign_message.rs

```diff
@@ -172,7 +179,11 @@ mod message_signing {
             pub fn from_base64(s: &str) -> Result<MessageSignature, MessageSignatureError> {
                 let bytes =
                     BASE64_STANDARD.decode(s).map_err(|_| MessageSignatureError::InvalidBase64)?;
-                MessageSignature::from_slice(&bytes)
+
+                // Convert the Vec<u8> into a fixed-length array
```

Comment:
I know, its literally there in code in the following line.  (I feeling argumentative this morning :)

Reply:
Sorry I missed this one when removing the first time, done now

Reply:
My bad, I was in a **** of a mood yesterday morning  - sometimes I shouldn't use the computer.